resources/conf/logback/logback-access.xml

```diff
@@ -1,18 +1,4 @@
 <?xml version="1.0"?>
-<!--
```

Comment:
We need to include this section if the default version of this file contains it. If the default omits it, then it's fine to remove as it might just be a carry over from Nexus 2.

Reply:
I checked the [default file](https://github.com/sonatype/nexus-public/blob/master/assemblies/nexus-base-template/src/main/resources/overlay/etc/logback/logback-access.xml) in the sonatype repo, seems they also removed this part.